ambry-frontend/src/test/java/com/github/ambry/frontend/FrontendRestRequestServiceTest.java

```diff
@@ -127,12 +128,14 @@
 public class FrontendRestRequestServiceTest {
   private final static QuotaManager QUOTA_MANAGER;
   private final static String CLUSTER_NAME = "ambry-test";
+  private final static QuotaChargeEventListener QUOTA_CHARGE_EVENT_LISTENER;
 
   static {
     try {
       QUOTA_MANAGER =
           new AmbryQuotaManager(QuotaTestUtils.createQuotaConfig(Collections.emptyMap(), false, QuotaMode.TRACKING),
               new MaxThrottlePolicy(), Mockito.mock(AccountService.class), null, new MetricRegistry());
+      QUOTA_CHARGE_EVENT_LISTENER = () -> {};
```

Comment:
we should definitely add new method in Router interface to avoid these empty callbacks.

Reply:
Using null now.